migrations/emailmonitor/oxa_email_monitor.py

```diff
@@ -0,0 +1,179 @@
+import MySQLdb
+import collections
+import os
+import re
+from datetime import datetime,timedelta
+
+
+def CreateYearMonthDay():
```

Comment:
Yep. I think I am using C# conventions and it looks like it doesn't fir the Python conventions.